tests/migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -266,9 +267,10 @@ sub run {
         # reboot
         send_key "alt-r";
         power_action('reboot', observe => 1, keepconsole => 1);
+        reconnect_mgmt_console if (is_pvm());
```

Comment:
please try to not put any backend specific code into test modules to make the top level of test code stating the clear test intention which is the same regardless of the backend: We want to reboot into a migrated system. If necessary put any necessary backend specific calls into the helper functions but at best the backend already handles any differences so that the test code is agnostic to the actual backend

Reply:
I don't understand 'backend specific code'. You mean 'is_pvm'?  I don't think this is backend code.

Reply:
You mean I should write one 'is_pvm' function at one lib file like lib/version_utils.pm. Same as 'is_sle' check?

Reply:
Well, it will not help much if you just move this code line into a new file in lib. The function `reconnect_mgmt_console` is already handling backend specific differences. See how the function `reconnect_mgmt_console` is currently used. The most prominent use is the test module
tests/boot/reconnect_mgmt_console.pm which is explicitly scheduled where needed. At best you can apply the same approach for yast2_migration.

Reply:
After reboot, only pvm cases need select console powerhmc-ssh. Then restart and go to grub page. Others don't need change console, just reconnect to root console. But when powerVM reboot, the root console will be deleted first, then select console powerhmc-ssh and rebuild one root console. So I need reconnect_mgmt_console to get root console back.

Reply:
I need add this is_pvm condition at here, Because for other platforms like x86 or aarch64 don't need reconnect_mgmt_console, and only for powerVM cases need reconnect_mgmt_console